### PR TITLE
feat: Add aggregate gene expression and cell counts for WMG V2 response

### PR DESCRIPTION
**Pull request by prathapsridharan**
## Reason for Change

- #4863 
- This PR addresses *step 6* in the ticket referenced. This PR concludes the work needed to make `\query` endpoint for WMG V2 functional (but not necessarily performant)

## Changes

- add aggregate gene expression stats to the response object returned by `\query` endpoint
- add aggregate cell counts stats to the response object returned by `\query` endpoint
- add unit tests to cover changes to the response object returned by `\query` endpoint

## Testing steps

- Unit tests

### Manual Tests on Rdev

1. Go to rdev for `/wmg/v2/query` endpoint: https://rdev-ps-wmg-v2-query-backend.rdev.single-cell.czi.technology/wmg/v2/ui/#/wmg/backend.wmg.api.v2.query
2. Paste in the following request body to the end point:

```
{"filter":{"dataset_ids":[],"development_stage_ontology_term_ids":[],"disease_ontology_term_ids":[],"gene_ontology_term_ids":["ENSG00000156738"], "organism_ontology_term_id":"NCBITaxon:9606","self_reported_ethnicity_ontology_term_ids":[],"sex_ontology_term_ids":[]},"is_rollup":true}
```

3. Verified that response contains aggregated gene expression for each `(gene_ontology_term_id, tissue_ontology_term_id)` combo like this:

```
        "aggregated": {
          "me": 1.953426840087639,
          "n": 570929,
          "tpc": 1.8715915423701033
        }
```

4. Verified that the response contains aggregated cell counts for each `tissue_ontology_term_id` like this:

```
        "aggregated": {
          "name": "respiratory basal cell (cell culture)",
          "tissue_ontology_term_id": "CL:0002633 (cell culture)",
          "total_count": 38665
        }
```

### CODE REVIEW

**Review by atarashansky — APPROVED** [review]
Fantastic work as always! This looks great. Approving now, but before merging could you link an `rdev` to the description so we could take a look?

File: backend/wmg/api/v2.py
Comment (prathapsridharan) [reply backend/wmg/api/v2.py:203]: @atarashansky @seve - I am going to merge this into `main`. The issue of the `tpc` is tracked in an issue and we can address it separately.

**Pull request merged**
